INSTALL

```diff
@@ -396,6 +396,10 @@ MFEM_USE_SIDRE = YES/NO
    blueprint specification. When enabled, this option requires installation of
    HDF5 (see also MFEM_USE_NETCDF), Conduit and LLNL's axom project.
 
+MFEM_USE_SIMD = YES/NO 
+   Enables the high performance templated classes to use specific intrinsics 
+   instead of the AutoSIMD (config/simd/auto.hpp) classe.
```

Comment:
Can't we enable the intrinsics automatically, based on compiler-defined macros like `__AVX__`? Or you just prefer to have an option to disable the auto-detection?

Reply:
Some detection is done in `tconfig.hpp`, but I thought it might be good to have the choice to use the `simd/auto.hpp` or the other `intrinsics` during some transition phase.

Reply:
`14s` vs `25s` on the mac `=>` it might be good to be the default.
Do we want to keep the `auto` as backup?

Reply:
What vs what is `14s` vs `25s`?

> it might be good to be the default.

Do you mean to make `MFEM_USE_SIMD = YES` the default?

> Do we want to keep the `auto` as backup?

I don't think we need to remove the default implementation of `AutoSIMD`.

Reply:
`done`